.editorconfig

```diff
@@ -74,6 +74,7 @@ csharp_style_expression_bodied_accessors = true:silent
 csharp_style_expression_bodied_lambdas = true:silent
 csharp_style_expression_bodied_local_functions = false:silent
 csharp_indent_labels = one_less_than_current
+csharp_style_prefer_primary_constructors = false:suggestion
```

Comment:
After discussion with Vemund whether to use primary constructors or not, we decided on not for now.